internal/pkg/service/appsproxy/proxy/proxy_test.go

```diff
@@ -1849,7 +1852,7 @@ func TestAppProxyRouter(t *testing.T) {
 
 			// Create a test server for the proxy handler
 			proxySrv := httptest.NewUnstartedServer(handler)
-			proxySrv.EnableHTTP2 = true
```

Comment:
HTTP2 `httptest` internalls don't log `superfluous response.WriteHeader ` errors, so I disabled it.